metrics/metric.py

```
@@ -0,0 +1,887 @@
+        #     url = url+f'$token={continue_token}'
+        # print(url)
+        if url in cache:
+            reply = cache[url]
+            if reply.status_code == 504:
+                session = requests.Session()
+                if s2api is not None:
+                    headers = {
+                        'x-api-key': s2api
+                    }
+                else:
+                    headers = None
```

Ruff (0.8.2)

Use ternary operator `headers = {'x-api-key': s2api} if s2api is not None else None` instead of `if`-`else`-block

Replace `if`-`else`-block with `headers = {'x-api-key': s2api} if s2api is not None else None`

(SIM108)